script.js

```diff
@@ -0,0 +1,34 @@
+const hamburger = document.getElementById('menuimg');
+const menu = document.getElementById('fullmenu');
+const names = document.getElementById('logonav');
+const closing = document.getElementById('closingbutton');
+const envelope = document.getElementById('envelope');
+const body = window.innerWidth;
+const item1 = document.getElementById('menuitemport');
+const item2 = document.getElementById('menuitemabout');
+const item3 = document.getElementById('menuitemcontact');
+
+const listener = () => {
+  item1.classList.add('active');
+  item2.classList.add('active');
+  item3.classList.add('active');
+  hamburger.classList.add('nodisplay');
+  envelope.classList.add('nodisplay');
+  menu.classList.add('response');
+  menu.style.display = 'block';
+  closing.classList.add('active');
+  names.classList.add('nodisplay');
+};
+
+hamburger.addEventListener('click', listener);
+
+if (body < 768) {
+  document.querySelectorAll('.menuitem').forEach((n) => n.addEventListener('click', () => {
+    menu.classList.remove('huevos');
+    closing.classList.remove('active');
```

Comment:
### Your Menu
![close-button (copy)](https://user-images.githubusercontent.com/93039359/179865877-c3598348-ab5e-4e94-a554-1c22d6628b63.png)

- [ ] When we click on the close button then it refreshes the page. This is not expected behavior. It should just make your menu hidden by clicking it. Please implement this change :blush: